web/modules/custom/helloworld/helloworld.info.yml

```diff
@@ -0,0 +1,9 @@
+name: Hello world
+type: module
+description: My custom Hello World module for Drupal 10.
+package: Custom Module
+version: 1.0
+core_version_requirement: ^10
+dependencies:
+  - drupal:views
+  - drupal:media
```

Comment:
Why are these depenencies required for the code that you have written?

Reply:
I was trying to see how dependencies work, hence I disabled it from the UI and made it required while installing the custom module to  see if it installs the required modules or not.